ProjectLighthouse.Servers.Website/Pages/UserPage.cshtml.cs

```diff
@@ -116,7 +157,8 @@ public async Task<IActionResult> OnGet([FromRoute] int userId)
 
         foreach (KeyValuePair<CommentEntity, RatedCommentEntity?> kvp in this.Comments)
         {
-            RatedCommentEntity? reaction = await this.Database.RatedComments.Where(r => r.CommentId == kvp.Key.CommentId)
+            RatedCommentEntity? reaction = await this.Database.RatedComments
```

Comment:
I understand that you prefer this style of indentation and I'm open to having a conversation about changing the entire project to this style, but for now, instead of changing them one by one we should just use the old style and then at a later date we can reformat the entire project